src/lucky/web_socket_handler.cr

```diff
@@ -0,0 +1,26 @@
+class WebSocketHandler
+  include HTTP::Handler
+
+  def call(context : HTTP::Server::Context)
+    if ws_route_found?(context) && websocket_upgrade_request?(context)
+      websocket_action.payload.new(context, websocket_action.params).websocket.call(context)
```

Comment:
I should probably rename these, but `websocket_action` is actually a `LuckyRouter::Match` and `payload` is the actual `Lucky::WebSocketAction` class. Then you call the `websocket` getter to call `HTTP::WebSocketHandler#call`.

One thing Lucky does here in a normal action is calls `perform_action` which checks that your route returns a `Lucky::Response` object. What should the websocket routes return? Or should they even return a thing?